src/components/HomePage/PrayerTimes/PrayerTimes.tsx

```diff
@@ -158,19 +156,9 @@ const PrayerTimes = () => {
   );
 };
 
-const useHijriDateFormatter = (hijriDate?: HijriDateData) => {
-  const { t } = useTranslation('home');
-  if (!hijriDate) return null;
-
-  const month = t(`hijri-date.month.${hijriDate.month}`);
-
-  // Different language have different format to show the date, so we need to "format" it.
-  // For example in Indonesia we say "12 Muharram 1443" instead of "Muharram 12, 1443"
-  return t('hijri-date.format', {
-    date: hijriDate.date,
-    month,
-    year: hijriDate.year,
-  });
+const useHijriDateFormatter = () => {
+  const { lang } = useTranslation('home');
+  return toLocalizedDate(new Date(), lang, { calendar: 'islamic', dateStyle: 'full' });
```

Comment:
We previously tried with local data too with 

```javascript
new Intl.DateTimeFormat('en-TN-u-ca-islamic', {day: 'numeric', month: 'long',weekday: 'long',year : 'numeric'}).format(Date.now());
```

If I remember correctly, i didn't show the right data. But it seems to be correct now. hmm. maybe need to double check this one